src/main/resources/templates/courses/assignments/group-pulls.ftl

```diff
@@ -37,7 +37,7 @@
                               </div>
                               <div class="pull-right"><i class="glyphicon glyphicon-comment"></i> ${pullRequest.comments?size} </div>
                             </span>
-                            <div class="comment">Pull Request #${pullRequest.issueId}: ${pullRequest.branchName}</div>
+                            <div class="comment">${pullRequest.title}</div>
```

Comment:
I was already afraid for that migrations, that's why it was commented out for so long. Do you want that migration in the changelog.xml?